src/audio/dai-zephyr.c

```diff
@@ -1203,7 +1203,8 @@ static int dai_copy(struct comp_dev *dev)
 	/* get data sizes from DMA */
 	ret = dma_get_status(dd->chan->dma->z_dev, dd->chan->index, &stat);
 	if (ret < 0) {
-		dai_report_xrun(dev, 0);
+		if (ret == -ENODATA)
```

Comment:
Shouldn't assume also -EPIPE according to https://www.alsa-project.org/alsa-doc/alsa-lib/pcm.html ?
New example in DMA:
https://github.com/zephyrproject-rtos/zephyr/pull/53327

Reply:
That does sound like a compelling argument, however here we don't have that error as a possible outcome from dma_get_status within SOF. Maybe a translation layer high above?